### ISSUE TITLE
cpp files is not recognized as and internal or external command error.

### ISSUE THREAD

**Issue by avijit1258** [kriscross07/atom-gpp-compiler#69]
'C:\Users\ovi\AppData\Local\Temp\11827-Maximum' is not recognized as an internal or external command,
operable program or batch file.

I have added MinGW/bin path to my environment to solve g++ not spawned problem.

**Comment by liangchunn** [comment]
I've had this problem before but I'm not too sure on how I fixed it.
The package is probably trying to open `11827-Maximum` and NOT `11287-Maximum.exe`

If you search in `C:\Users\ovi\AppData\Local\Temp\` you should find the .exe file there

**Comment by avijit1258** [comment]
yes package is trying to open the exe from your defined location but without .exe . But i need help .

**Comment by hsensh** [comment]
same here it's not recognizing the full path it stops on a space or on a number and doesn't continue
ie: my path would be C:\users\desktop\folder1\folder2\file.exe
it would read it C:\users\desktop\folder1\

**Comment by heronova149** [comment]
I, as well, have the same problem. I have looked at a dozen of different sites and forums but haven't found any solution to this. 
![image](https://user-images.githubusercontent.com/34384962/33788413-5bce7f1e-dc72-11e7-9083-9513bb38db85.png)
Any help would be appreciated. :)

**Comment by mda6** [comment]
Removing the spaces from my location of my project folder helped me fix this problem

**Comment by heronova149** [comment]
Thanks it worked perfectly.

**Comment by romanpoudel** [comment]
Thank you very much
.

**Comment by K-Pranay** [comment]
I found a solution for this 
1. open gpp compiler settings in atom.(packages->settings view->manage packages)
2. you will find gpp compiler settings,open that and make sure that if compile to temporary  directory is uncheck.
3. this will work even if you have spaces in user name.
4. if this does not work their might be problem in mingw installation and its path.

**Comment by bartlomiej-stefanski** [comment]
It was super weird for me. My file with name structure "sometext (othertext).cpp" didn't compile. After changing it's name to "a.cpp", compiling and then going back to original name it suddenly started to work

**Comment by h4jack** [comment]
Please tell me how to fix that 
![Screenshot 2023-08-21 152458](https://github.com/kriscross07/atom-gpp-compiler/assets/130727297/c2e47548-6a57-4760-be86-35dce2d2931a)
